pkg/scheduler/actions/allocate/allocate.go

```diff
@@ -247,6 +247,7 @@ func (alloc *Action) Execute(ssn *framework.Session) {
 			stmt.Commit()
 		} else {
 			stmt.Discard()
+			break
```

Comment:
One question: if break here, then it will not try to schedule other jobs in the same period?

Reply:
yes.